lib/hubspot/contact.rb

```diff
@@ -163,6 +163,14 @@ def merge!(primary_contact_vid, secondary_contact_vid)
           body: { vidToMerge: secondary_contact_vid }
         )
       end
+
+      # Updates the properties of a contact
+      # {https://developers.hubspot.com/docs/methods/contacts/update_contact}
+      # @param params [Hash] hash of properties to update
+      def update!(vid, params = {})
+        query = {"properties" => Hubspot::Utils.hash_to_properties(params.stringify_keys!)}
+        Hubspot::Connection.post_json(UPDATE_CONTACT_PATH, params: { contact_id: vid }, body: query)
```

Comment:
This is not consistent with the other added `update!()` methods in that it doesn't return a new instance of the updated resource. This should return a new instance of the object.

Reply:
The problem is that this endpoint
> Returns a 204 No Content response on success.

https://developers.hubspot.com/docs/methods/contacts/update_contact

So if you want to return a new instance, you'd have to fetch it, which makes two calls instead of one.

Reply:
@fonji Good point. For reasons like this, I think we are headed in the direction of always returning a `Hubspot::Response` object instead of attempting to return an instance of the class. PR #171 moves us in that direction, starting with some of the `update` actions. Once @cbisnett has time to review and approve this approach (and given it's the holidays, that may not be until after the new year) we can continue updating all the actions to either `raise` or return a `Hubspot::Response`, with the goal of releasing these changes as part of the `v1` release.